### PR TITLE
Updated Tesseract to 4.1.1.

### PR DESCRIPTION
**Pull request by pixel27**
I disabled -ffast-math to conform with current requirements.  Unfortunately it doesn't appear you can disable it with a compile flag so I put together a patch to apply to the make file.

### CODE REVIEW

File: T/Tesseract/build_tarballs.jl [hunk 1]
```diff
@@ -3,12 +3,12 @@
...
 name = "Tesseract"
-version = v"4.1.0"
+version = v"4.1.1"
 
 # Collection of sources required to build Tesseract
 sources = [
     "https://github.com/tesseract-ocr/tesseract/archive/$(version).tar.gz" =>
-    "5c5ed5f1a76888dc57a83704f24ae02f8319849f5c4cf19d254296978a1a1961",
+    "2a66ff0d8595bff8f04032165e6c936389b1e5727c3ce5a27b3e059d218db1cb",
     "./bundled"
```
Comment (giordano) [inline T/Tesseract/build_tarballs.jl:12]: Can you please the new types [`ArchiveSource`](https://juliapackaging.github.io/BinaryBuilder.jl/dev/reference/#BinaryBuilder.ArchiveSource) and [`DirectorySource`](https://juliapackaging.github.io/BinaryBuilder.jl/dev/reference/#BinaryBuilder.DirectorySource)?  The same also for the dependency, use [`Dependency`](https://juliapackaging.github.io/BinaryBuilder.jl/dev/reference/#BinaryBuilder.Dependency)

Comment (pixel27) [reply T/Tesseract/build_tarballs.jl:12]: Just to confirm, the easiest way to do that would be to run the BinaryBuilder.run_wizard() then merge in the appropriate lines from what it generates?

Comment (giordano) [reply T/Tesseract/build_tarballs.jl:12]: No, the easiest way is just to wrap `ArchiveSource` around the first source (and replace `=>` with `,`), `DirectorySource` around the second one, and `Dependency` around all dependencies

Comment (giordano) [reply T/Tesseract/build_tarballs.jl:12]: Like this: https://github.com/JuliaPackaging/Yggdrasil/pull/1006/files

Comment (pixel27) [reply T/Tesseract/build_tarballs.jl:12]: Okay, thanks, will do.

**Pull request merged**